content/js/users.md

```diff
@@ -1,40 +1,59 @@
 <a name="manage-user"></a>
 ## Manage User
 
+### How does it work
+
+Notice that even without logging in, user can still interact with the SDK,
+such as querying data in `skygear.publicDB`. However, this behavior can be
+changed by setting different access control for public. For more information,
+please refer to [access control](/js/guide/access-control#acl-default) page.
+Skygear uses `accessToken` with localstorage to track user authentication.
```

Comment:
I think access token should be written as is (not as code) throughout. If you want to illustrate as code, you can add the code in parenthesis after the description